admin_guide/topics/creating-golden-images-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='creating-golden-image-cluster-auto-scaler-{context}']
+= Creating prime image (aka golden images)
+
+[NOTE]
+====
+If you already have a pre-existing primed image you can skip this
+section.
+====
+
+You can use Ansible playbooks to automatically create and seal a
+primed image that is suitable for use by a LC and its ASG. All the
+attibutes in the following steps are from your existing AWS cluster.
+
+.Procedure
+
+. Create an Ansible *_inventory_* file
++
+----
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+openshift_deployment_type=openshift-enterprise
+ansible_ssh_user=ec2-user
+openshift_clusterid=mycluster
+ansible_become=yes
+
+[masters]
+[etcd]
+[nodes]
+----
+
+. Create provisioning file *_build-ami-provisioning-vars.yaml_*
++
+[source,yaml]
+----
+openshift_deployment_type: openshift-enterprise <1>
+
+openshift_aws_clusterid: mycluster <2>
+
+openshift_aws_region: us-east-1 <3>
+
+openshift_aws_create_vpc: false <4>
+
+openshift_aws_vpc_name: production <5>
+
+openshift_aws_subnet_az: us-east-1d <6>
+
+openshift_aws_create_security_groups: false <7>
+
+openshift_aws_ssh_key_name: production-ssh-key <8>
+
+openshift_aws_base_ami: ami-12345678 <9>
+
+openshift_aws_create_s3: False <10>
+
+openshift_aws_build_ami_group: default <11>
+
+openshift_aws_vpc: <12>
+  name: "{{ openshift_aws_vpc_name }}"
+  cidr: 172.18.0.0/16
+  subnets:
+    us-east-1:
+    - cidr: 172.18.0.0/20
+      az: "us-east-1d"
+
+ifdef::openshift-enterprise[]
+rhsub_user: user@example.com <13>
+rhsub_pass: password <14>
+rhsub_pool: pool-id <15>
+endif::[]
+----
+<1> The type of deployment
+<2> The name of the existing cluster
+<3> The region the existing cluster is currently running in
+<4> Disable the creation of a VPC
+<5> The existing VPC name that the cluster is running in
+<6> The name of one subnet the existing cluster is running in
+<7> Disable the creation of security groups
```

Comment:
The security group is what is used in your currently running cluster.

We're leveraging the build_ami playbooks; ordinarily they would create everything from scratch. In this case, as we already have a working cluster, we just need to dig out and capture these variables so that the playbook does not go and create these components (VPC, Security Groups, etc) from scratch.